scenario/README

```diff
@@ -0,0 +1,24 @@
+By default CloudFerry tool is going to migrate all resources and instances from the source cloud to the destination cloud.
```

Comment:
we have plenty in CF, for example, our primary readme uses markdown: https://github.com/MirantisWorkloadMobility/CloudFerry/blob/master/README.md